net-im/libcommuni/libcommuni-3.5.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit qmake-utils
+
+DESCRIPTION="A cross-platform IRC framework written with Qt"
+HOMEPAGE="http://communi.github.io/"
+SRC_URI="https://github.com/communi/libcommuni/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="benchmarks debug examples icu qml static tests +uchardet"
+
+# Compiling with USE="examples" or USE="tests" is currently broken with dev-libs/icu
+REQUIRED_USE="^^ ( icu uchardet )
+		examples? ( uchardet )
+		tests? ( uchardet )"
+
+RDEPEND="dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	icu? ( dev-libs/icu )
```

Comment:
`:=`, this thing breaks ABI every other week ;-).

Reply:
If I understand this correctly, ```:=``` means, when the lib changed (e.g. new slot version), an automatic rebuild of this package will be requested by portage?

Reply:
Yes, if slot or subslot changes.